src/views/MainView.vue

```diff
@@ -60,13 +60,17 @@ export default Vue.extend({
     }
   },
   computed: {
-    ...mapState(['toast'])
+    ...mapState(['toast', 'isSignedIn'])
   },
   methods: {
-    ...mapMutations(['clearToast']),
+    ...mapMutations(['clearToast', 'setToast']),
     ...mapActions(['load', 'loadVariables', 'loadAssessments'])
   },
   async created () {
+    if (!this.isSignedIn && !this.toast) {
+      console.log('call set toast')
+      this.setToast({ type: 'info', message: 'Please sign in to select and order variables' })
```

Comment:
Toast doesn't show when I click on an item, although I see the 'call set toast' message.

Reply:
The toast should be shown when not signed in , i do see the toast at the top of the screen 
<img width="424" alt="Screenshot 2019-11-06 at 18 09 00" src="https://user-images.githubusercontent.com/47183404/68320623-957dec80-00c0-11ea-995c-7266e8851a94.png">


suggest we have a look tomorrow